sdk/tests/network-client.test.ts

```diff
@@ -357,7 +357,7 @@ describe('NodeConnection', () => {
                 expect(transactions.length).equal(4);
                 expect(transactions[0].status).equal("accepted");
                 expect(transactions[0].type).equal("execute");
-                expect(transactions[0].index).equal(0);
+                expect(transactions[0].index).equal(BigInt(0));
```

Comment:
You don't need to make any changes, but just for future reference you can use `0n` for bigint literals, which makes them a lot more convenient to use.

Reply:
That's a good tip, thanks.